x-pack/legacy/plugins/rollup/__jest__/client_integration/job_list_clone.test.js

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
so the reason they're in two different files is because they're testing two different components (both files, on line 25, yank the either setup from `job_create.helpers.js` or `job_list.helpers.js`. Not sure if that makes sense?

Reply:
it looks like they're both using this:

```
const { setup } = pageHelpers.jobList;
```
is there something else i'm missing?

Reply:
sorry if my explanation was a bit confusing 🙂 - I may also be missing something:

`job_create_clone.test.js` is using [this](https://github.com/elastic/kibana/pull/41891/files#diff-fbb92028a0e577a534c9e34b75633ebdR25) setup which creates a testbed for the `JobCreate` component.

vs.

`job_list_clone.test.js` is using [this](https://github.com/elastic/kibana/pull/41891/files#diff-2368af1324c141126c69c3442fd27330R25) setup which creates a testbed for the `JobList` component.

Because of the disconnect in the mocked out router the test doesn't actually navigate from list to create form (which is a weak point). Perhaps @sebelga could assist a bit with how we could best get around this?

P.S.
When opening those links beware the delay before the jump-to-line happens 👽

Reply:
Ah - I missed his comment below. Will take a look.

Reply:
Ah, I think I see the confusion now. My comment was regarding the `job_list_clone.test` and `job_list.test` files (not `job_create_clone.test`). Does that make sense now?

Reply:
Yep - sorry I missed this!